skills/setup/SKILL.md

```
@@ -212,26 +227,36 @@ Use AskUserQuestion to ask ONLY questions relevant based on detection. 3-5 quest
+    PLUGIN_ROOT="$(cd "$(dirname "$setup_script")/.." && pwd)"
+  fi
+fi
+if [ -z "$PLUGIN_ROOT" ] && [ -f "$(pwd)/lib/setup-mandatory-files.sh" ]; then
+  PLUGIN_ROOT="$(pwd)"
+fi
+bash "${PLUGIN_ROOT}/lib/setup-mandatory-files.sh" "$(pwd)" <threshold> "<coverage-command>" --platform <platform>
 ```
 
-Example for Python with pytest at 100%:
+Example for Codex with Python/pytest at 100%:
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)